src/runtime/hexagon_remote/dlib.cpp

```diff
@@ -330,7 +341,14 @@ struct dlib_t {
                 return false;
             }
         }
-
+        if (!fini) {
+            // This is not an error.
+            log_printf("DT_FINI not defined\n");
```

Comment:
Keep in mind log_printf prints in all circumstances do adb logcat, so this message is going to get printed always. I think these prints should be removed, I can see why they were useful during debugging, but I don't think there should be unconditional prints in non-error cases unless really needed (I don't think we have any right now).